README.md

```diff
@@ -80,6 +86,23 @@ public function __construct(
 
 ## Code structure
 
+### Use statments
+
+Group `use` statements as follows:
+
+- Custom/contrib modules
+- Core modules
+- Php base modules
+
+```
+use Drupal\my_module\Entity;
+use Drupal\commerce_product\Entity;
+
+use Drupal\Core\User;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+```
+
```

Comment:
Can we add that this is mandatory only if there are more than 5 use statements.

Reply:
Let me rewrite this section with more information:

### Use statements

Group `use` statements to improve readability. This is mandatory only if there are more than 5 `use` statements. Grouping should happen in the following order:

- Drupal modules
- Drupal core
- External libraries

```
use Drupal\my_module\Entity\MyEntity;
use Drupal\commerce_product\Entity\ProductInterface;
use Drupal\user\UserInterface;

use Drupal\Core\Entity\EntityTypeManagerInterface;

use Symfony\Component\DependencyInjection\ContainerInterface;
```

All statements within each group should follow alphabetical order. If the Drupal modules group grows really large, it can be further broken down in the following groups:

- Custom modules
- Contrib modules
- Core modules

```
use Drupal\my_module\Entity\MyEntity;
use Drupal\my_module\Entity\Storage\MyEntityStorageInterface;

use Drupal\commerce_product\Entity\ProductInterface;
use Drupal\commerce_product\Entity\ProductVariationInterface;
use Drupal\commerce_order\Entity\OrderInterface;

use Drupal\user\UserInterface;
use Drupal\taxonomy\TermInterface;
```